internal/policies/apparmor/apparmor.go

```diff
@@ -229,17 +236,95 @@ func (m *Manager) ApplyPolicy(ctx context.Context, objectName string, isComputer
 	if err := os.RemoveAll(oldApparmorPath); err != nil {
 		return fmt.Errorf(i18n.G("can't remove old apparmor directory %q: %v"), oldApparmorPath, err)
 	}
+	return nil
+}
+
+// applyUserPolicy applies apparmor policies for the user object.
+func (m *Manager) applyUserPolicy(ctx context.Context, e entry.Entry, apparmorPath string, username string, assetsDumper AssetsDumper) (err error) {
+	defer decorate.OnError(&err, i18n.G("can't apply user policy"))
+
+	// Create a temporary filepath to be used by the assets dumper
```

Comment:
The line below still doesn’t create a temp dir :p
What you can do, it just put that all in the same stenza (no line 248 + merge with comment on line 249)